modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/script/library/v1/ScriptLibrarySuccessTest.java

```diff
@@ -0,0 +1,179 @@
+                        SCRIPT_LIBRARY_DESCRIPTION);
+        responseOfUpload.then()
+                .log().ifValidationFails()
+                .assertThat()
+                .statusCode(HttpStatus.SC_CREATED)
+                .header(HttpHeaders.LOCATION, notNullValue());
+        String location = responseOfUpload.getHeader(HttpHeaders.LOCATION);
+        assertNotNull(location);
+    }
+
+    @Test(dependsOnMethods = {"testAddScriptLibrary"})
+    public void testGetScriptLibrary() throws XPathExpressionException {
+
+        SCRIPT_LIBRARY_CONTENT_REF = "/t/" + context.getContextTenant()
+                .getDomain() + "/api/server/v1/script-libraries/" + SCRIPT_LIBRARY_NAME + "/content";
+        Response response = getResponseOfGet(SCRIPT_LIBRARY_API_BASE_PATH + PATH_SEPARATOR + SCRIPT_LIBRARY_NAME);
+        response.then()
+                .log().ifValidationFails()
+                .assertThat()
+                .statusCode(HttpStatus.SC_OK)
+                .body("name", equalTo(SCRIPT_LIBRARY_NAME))
+                .body("description", equalTo(SCRIPT_LIBRARY_DESCRIPTION))
+                .body("content-ref", equalTo(SCRIPT_LIBRARY_CONTENT_REF));
+    }
+
+    @Test(dependsOnMethods = {"testAddScriptLibrary"})
+    public void testGetScriptLibraries() {
+
+        Response response = getResponseOfGet(SCRIPT_LIBRARY_API_BASE_PATH);
+        response.then()
+                .log().ifValidationFails()
+                .assertThat()
+                .statusCode(HttpStatus.SC_OK);
+    }
+
+    @Test(dependsOnMethods = {"testGetScriptLibrary"})
+    public void testUpdateScriptLibrary() throws IOException, XPathExpressionException {
+
+        SCRIPT_LIBRARY_CONTENT_REF = "/t/" + context.getContextTenant()
+                .getDomain() + "/api/server/v1/script-libraries/" + SCRIPT_LIBRARY_NAME + "/content";
+        String content = readResource("sample-script-library-updated.js");
+        String endpoint = SCRIPT_LIBRARY_API_BASE_PATH + PATH_SEPARATOR + SCRIPT_LIBRARY_NAME;
+        Response response = getResponseOfMultipartPut(endpoint, content, SCRIPT_LIBRARY_UPDATED_DESCRIPTION);
+        response.then()
+                .log().ifValidationFails()
+                .assertThat()
+                .statusCode(HttpStatus.SC_OK)
+                .body("name", equalTo(SCRIPT_LIBRARY_NAME))
+                .body("description", equalTo(SCRIPT_LIBRARY_UPDATED_DESCRIPTION))
+                .body("content-ref", equalTo(SCRIPT_LIBRARY_CONTENT_REF));
+    }
+
+    @Test(dependsOnMethods = {"testUpdateScriptLibrary"})
+    public void testGetContentScriptLibrary() throws IOException {
+
+        String content = readResource("sample-script-library-updated.js");
+        String endpoint =
+                SCRIPT_LIBRARY_API_BASE_PATH + PATH_SEPARATOR + SCRIPT_LIBRARY_NAME + SCRIPT_LIBRARY_CONTENT_PATH;
+        Response response = getResponseOfGet(endpoint, "application/octet-stream");
```

Comment:
shall we define a contant for "application/octet-stream"

Reply:
[Fixed commit](https://github.com/wso2/product-is/pull/7338/commits/3e4452fb2e570a8cb1dc3855e6837b7e264dfb91)